main.tf

```diff
@@ -1,24 +1,36 @@
 /**
- * Creates and configures an S3 bucket for storing logs from various AWS
- * services and enables CloudTrail on all regions. Logs will expire after a
- * default of 90 days. Includes support for sending CloudTrail events to a
- * CloudWatch Logs group.
+ * Supports two main uses cases:
  *
- * Logging from the following services is supported:
+ * 1. Creates and configures a single private S3 bucket for storing logs from various AWS services, which are nested as bucket prefixes, and enables CloudTrail on all regions. Logs will expire after a default of 90 days, with option to configure retention value. Includes support for sending CloudTrail events to a CloudWatch Logs group.
+ *
+ * 2. Creates and configures a single private S3 bucket for a single AWS service, and enables CloudTrail on all regions. Logs will expire after a default of 90 days, with option to configure retention value. Includes support for sending CloudTrail events to a CloudWatch Logs group.
```

Comment:
Numbered lists shouldn't have a newline between them and should all start with `1.`.

Reply:
fixed